api/envoy/api/v2/discovery.proto

```diff
@@ -93,3 +93,92 @@ message DiscoveryResponse {
   // required for non-stream based xDS implementations.
   string nonce = 5;
 }
+
+// IncrementalDiscoveryRequest and IncrementalDiscoveryResponse are used in a
+// new GRPC endpoint for Incremental XDS. The feature is not supported for REST
+// management servers.
+//
+// With Incremental XDS, the IncrementalDiscoveryResponses do not need to
+// include a full snapshot of the tracked resources. Instead
+// IncrementalDiscoveryResponses are a diff to the state of a XDS client.
+// In incremental XDS there are per resource versions which allows to track
+// state at the resource granularity.
+// An XDS Incremental session is always in the context of a GRPC bidirectional
+// stream. This allows the XDS server to keep track of the state of XDS clients
+// connected to it.
+//
+// In Incremental XDS the nonce field is required and used to pair
+// IncrementalDiscoveryResponse to a IncrementalDiscoveryRequest ACK or NACK.
+// A message level system_version_info is present for debugging purposes only.
```

Comment:
Is the intention here to try to keep implementations from relying on Envoy setting this value in requests?

I think this documentation should just describe the client behavior (e.g., the client returns the value it saw in the last ACKed response) and leave off the "debugging purposes only" statements.

Reply:
I updated the comment to match having it in responses only.